Gemfile

```diff
@@ -9,6 +9,7 @@ end
 
 gem "active_record_union"
 gem "activerecord-import"
+gem "auto_strip_attributes"
```

Comment:
Do we need a gem for this? Can we simply define `def facility=()`?

Reply:
I think so, I'd also tried that and it works. The only difference between this gem and setters is that:

* It does a `before_save` hook rather than an attribute set.
* You can easily write a whole bunch of generic filters across many attributes (like facility name, org name, protocol name etc.).

I'm totally fine with doing it either way. Just felt that there was some value to this really tiny gem.

Reply:
I'm actually a fan of overriding the attribute set 😅 But I can see how we can apply the "titleize" filter across several attributes easily. To avoid YAGNI though, I'd like to set up the simple attr method, and come straight back to this gem if we need to do this across several attributes. What do you think? Less tooling for us to wrap our heads around :)

Reply:
@harimohanraj89 I added some more basic squishes to more editable fields within this PR, because why not...